cassandra-migration/src/main/java/org/cognitor/cassandra/migration/Database.java

```diff
@@ -273,7 +266,7 @@ boolean takeLeadOnMigrations(int repositoryLatestVersion) {
                 LOGGER.debug("Trying to take lead on schema migrations");
                 BoundStatement boundStatement = takeMigrationLeadStatement.bind(getKeyspaceName(), this.instanceId,
                         this.instanceAddress);
-                ResultSet lwtResult = session.execute(boundStatement);
+                ResultSet lwtResult = executeStatement(boundStatement);
```

Comment:
As mentioned in the comment of the other PR. Lets keep this with Quorum. I will take care of this after the merge.